cmd/kubelet/app/server.go

```diff
@@ -232,8 +232,7 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 			logs.InitLogs()
 			logOption := &logs.Options{Config: kubeletConfig.Logging}
 			if err := logOption.ValidateAndApply(utilfeature.DefaultFeatureGate); err != nil {
-				klog.ErrorS(err, "Failed to initialize logging")
-				os.Exit(1)
+				return fmt.Errorf("failed to initialize logging: %w", err)
```

Comment:
This was part of https://github.com/kubernetes/kubernetes/commit/4df70dfd41601077478289ad358e7c10a4f0d0e1

What has changed to remove it?

Reply:
The logic is the same, except that the upper layer performs log output and exit. @aojea WDTF?

Reply:
WDTF?? or WDTY?? :smile:

Reply:
WDTY got it! 😄😄😄😄😄😄